src/RewardsStreamerMP.sol

```diff
@@ -479,17 +496,21 @@ contract RewardsStreamerMP is
         return deltaMP;
     }
 
-    function _updateVaultMP(address vaultAddress, bool forceMPUpdate) internal {
+    function _updateVault(address vaultAddress, bool forceMPUpdate) internal {
         VaultData storage vault = vaultData[vaultAddress];
         uint256 accruedMP = _getVaultPendingMP(vault);
         if (accruedMP > 0 || forceMPUpdate) {
             vault.mpAccrued += accruedMP;
             vault.lastMPUpdateTime = block.timestamp;
         }
+
+        uint256 rewardsAccrued = _vaultPendingRewards(vault);
+        vault.rewardsAccrued += rewardsAccrued;
+        vault.rewardIndex = rewardIndex;
     }
 
     function updateVaultMP(address vaultAddress) external onlyNotEmergencyMode {
```

Comment:
I guess there's a reason you've renamed `_updateValtMP` to `_updateVault`.
Should we also rename` updateVaultMP` to `updateVault`?